<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgIkConditional</class>
 <widget class="QDialog" name="CQDlgIkConditional">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IK Group Conditional Parameters</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Perform if ...</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="qqIkGroupCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="qqIkResultCombo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Restore if ...</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqFailedLinear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Linear precision requirements were not fullfilled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqFailedAngular">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Angular precision requirements were not fullfilled</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="qqClose">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>150</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqIkGroupCombo</tabstop>
  <tabstop>qqIkResultCombo</tabstop>
  <tabstop>qqFailedLinear</tabstop>
  <tabstop>qqFailedAngular</tabstop>
  <tabstop>qqClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
